apps/next/src/components/TokenSelect/TokenSelect.tsx

```diff
@@ -20,6 +20,8 @@ type TokenSelectProps = {
   query: string;
   onQueryChange: (query: string) => void;
   hint?: string;
+  disabled?: boolean;
+  tokenBalance?: boolean;
```

Comment:
Can we rename it to `withTokenBalance` or `showTokenBalance` instead? Just so it's not confused with the actual balance. Same in `SelectedToken.tsx`.